docs/user/index.rst

```diff
@@ -244,7 +245,8 @@ of Read the Docs and the larger software documentation ecosystem.
   :doc:`security` |
   :doc:`DMCA takedown policy <dmca/index>` |
   :doc:`Policy for abandoned projects <abandoned-projects>` |
-  :doc:`Release notes & changelog <changelog>`
+  :doc:`Release notes & changelog <changelog>` |
```

Comment:
Looks like the terminating `|` is extranous, without a following item in the list.

```suggestion
  :doc:`Release notes & changelog <changelog>`
```